### PR TITLE
Update imports handling

### PR DESCRIPTION
**Pull request by TobKed**

### CODE REVIEW

File: airflow_munchkin/block_generator/hook_generator.py [hunk 1]
```diff
@@ -103,7 +102,7 @@ def convert_path_parameter_block_to_individual_parameters(
     required_parameters: Dict[str, ParameterBlock] = {}
     call_args = []
 
-    for name in path_info.args:
+    for name in path_info.args:  # type: ignore
```
Comment (mik-laj) [inline airflow_munchkin/block_generator/hook_generator.py]: We don't use a `type: ignore` in this project. Can you fix type hint instead?

File: airflow_munchkin/block_generator/hook_generator.py [hunk 2]
```diff
@@ -72,7 +72,7 @@ def generate_get_conn_method_block(integration: Integration) -> MethodBlock:
     return method_block
 
 
-PATH_REGEXP = re.compile(r"``(?P<path>(?:[a-z{}_ ]+\/[a-z{}_ ]+)+)``")
+PATH_REGEXP = re.compile(r"`\"?(?P<path>(?:[a-z{}_ ]+\/[a-z{}_ ]+)+)\"?`")
```
Comment (mik-laj) [inline airflow_munchkin/block_generator/hook_generator.py]: Can you add tests to check this case?

File: airflow_munchkin/block_generator/hook_generator.py [hunk 3]
```diff
@@ -83,9 +83,8 @@ def find_matching_path_info(path: str, path_infos: Dict[str, PathInfo]) -> PathI
     for path_info in path_infos.values():
         if path_info.args == arguments:
             return path_info
-    raise GeneratorException(
-        f"The path could not be determined. Current path: '${path}'"
-    )
+
+    return PathInfo(name="TODO", args=arguments)
```
Comment (mik-laj) [inline airflow_munchkin/block_generator/hook_generator.py]: Can you add tests to check this case?

File: airflow_munchkin/client_parser/cloud_client_parser.py [hunk 4]
```diff
@@ -64,7 +64,7 @@ def parse_client(clazz_name: str) -> ClientInfo:
     functions = inspect.getmembers(clazz, predicate=inspect.isfunction)
     path_methods = {
         name: parse_path_method(name, method)
-        for name, method in methods
+        for name, method in methods + functions
```
Comment (mik-laj) [inline airflow_munchkin/client_parser/cloud_client_parser.py]: Can you add tests to check this case?

File: airflow_munchkin/block_generator/hook_generator.py (hunk 1 above)
Comment (TobKed) [reply airflow_munchkin/block_generator/hook_generator.py]: Sure, I will fix it.

File: airflow_munchkin/block_generator/hook_generator.py (hunk 3 above)
Comment (TobKed) [reply airflow_munchkin/block_generator/hook_generator.py]: Test added.

File: airflow_munchkin/client_parser/cloud_client_parser.py (hunk 4 above)
Comment (TobKed) [reply airflow_munchkin/client_parser/cloud_client_parser.py]: Existing test extended slightly.

File: airflow_munchkin/block_generator/hook_generator.py (hunk 1 above)
Comment (TobKed) [reply airflow_munchkin/block_generator/hook_generator.py]: Moreover, during deletion of mine "ignores" I also deleted some additional `type: ignore` which seems to be unnecessary  🕺

**Comment by TobKed** [comment]
cc @mik-laj @nuclearpinguin

**Pull request merged**